tests/test_text.py

```diff
@@ -29,12 +29,20 @@ def test_parse_yaml():
     assert set(o) == {"a", "b"}
 
 
-@pytest.mark.parametrize("yaml_url, expected", [("https://raw.githubusercontent.com/GeoscienceAustralia/dea-config/"\
-                                      "master/dev/services/odc-stats/fc_percentile/ga_ls_fc_pc_cyear_3.yaml",
-                                      "ga_ls_fc_pc_cyear_3")])
-def test_load_yaml_remote(yaml_url, expected):
-    content = load_yaml_remote(yaml_url)
-    assert content["product"]["name"] == expected
+def test_load_yaml_remote(httpserver):
+    httpserver.expect_request("/ga_ls_fc_pc_cyear_3.yaml").respond_with_data("""
+    a: 3
+    b: foo
+    """
+    )
+    content = load_yaml_remote(httpserver.url_for("/ga_ls_fc_pc_cyear_3.yaml"))
+    assert content["a"] == 3 and content["b"] == "foo"
+    try: 
+        content = load_yaml_remote(httpserver.url_for("/something_non_exist.yaml"))
+    except Exception as e:
+        pass
+    else:
+        assert False
```

Comment:
This is not the right way to check that an exception is raised.

See: https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions

Reply:
thanks! fixed.